src/MassMatrix3_TEST.cc

```diff
@@ -305,6 +308,9 @@ TEST(MassMatrix3dTest, PrincipalMoments)
     EXPECT_EQ(m.PrincipalMoments(), Ieigen);
     EXPECT_TRUE(m.IsPositive());
     EXPECT_TRUE(m.IsValid());
+    // Test with different tolerances
+    EXPECT_EQ(m.PrincipalMoments(0.24), Ieigen);
+    EXPECT_EQ(m.PrincipalMoments(0.26), Ixxyyzz);
```

Comment:
These 2 numbers look very similar, is there a reason why both are interesting to test? It would be nice to mention on the comment. The same for the new lines above.

Reply:
I've expanded the comments in https://github.com/ignitionrobotics/ign-math/pull/424/commits/40fe3c471914cd7ecafe09362307e8e5b464a556; is it clearer now?

Reply:
Very helpful, thanks!